pom.xml

```diff
@@ -41,6 +41,8 @@
         <module>nifi-toolkit</module>
         <module>nifi-manifest</module>
         <module>c2</module>
+        <module>nifi-h2-database-migrator</module>
+        <module>nifi-h2-database-updater</module>
```

Comment:
What do you think about either moving these under their own module like `nifi-h2`, or moving them under `nifi-commons`?